cypress/integration/functional/wallet/settings/privacyLock_with_stub.spec.ts

```diff
@@ -0,0 +1,39 @@
+// import * as LocalAuthentication from 'expo-local-authentication'
+
+context.skip('Wallet - Privacy Lock', () => {
+  before(function () {
+    /**
+     * FIXME:
+     * stub LocalAuthentication require recompile
+     * error: `require appropriate loader` when local authentication import direct from react-native
+     */
+    // cy.stub(LocalAuthentication, 'hasHardwareAsync').returns(true)
```

Comment:
@thedoublejay this is what I mentioned to you earlier.

When import + stub happened within the cypress main, it will cause some changes to the compiling process. Few errors I bumped into and haven't managed to solve them. Possibly easy solve with the correct cypress compiler option, but will takes me time learn through the cypress doc and some trial and error.

Reply:
![image](https://user-images.githubusercontent.com/44192990/130601885-b6e7f860-5e70-48a3-a3e0-92477b2b9829.png)